facades/PC/README.md

```diff
@@ -0,0 +1,99 @@
+# PC Facade
+
+The _PC Facade_ is the front-facing layer for running _Terasology_ on the PC.
+
+## Usage
+
+Most users won't interact directly with the PC Facade, but instead use the [Terasology Launcher](https://github.com/MovingBlocks/TerasologyLauncher) to start the game.
+
+When starting distributions of Terasology manually, users will interact with the command-line interface ([CLI](#cli)).
+
+Developers will most often use the PC Facade via [Gradle](#gradle) or via a run configuration in their IDE.
+
+### CLI
+
+The PC Facade provides a command-line interface (CLI) for configuring Terasology.
+See the usage help (`./Terasology --help`) for a full overview.
```

Comment:
Is it worth mentioning that the CLI options are managed through the [`Terasology`](https://github.com/MovingBlocks/Terasology/blob/26d5b5237ba34b4b4c97d9f10469512432b14d90/facades/PC/src/main/java/org/terasology/engine/Terasology.java) class using [PicoCLI](https://picocli.info/)?

Reply:
Yeah why not... How about the following:
```suggestion
See the usage help (`./Terasology --help`) for a full overview.
For implementing the CLI, we make use of the [PicoCLI](https://picocli.info/) framework for Java command-line applications.
```

Reply:
That wording works fine. It's just a bit of useful context in-case somebody wanted to know how to add a new command line option, for example.